src/kernel/mem.zig

```diff
@@ -29,3 +39,55 @@ pub fn init(mb_info: *multiboot.multiboot_info_t) MemProfile {
         .fixed_alloc_size = FIXED_ALLOC_SIZE,
     };
 }
+
+///
+/// Convert a virtual address to its physical counterpart by subtracting the kernel virtual offset from the virtual address.
+///
+/// Arguments:
+///     IN virt: var - The virtual address to covert. Either an integer or pointer.
+///
+/// Return: @typeOf(virt)
+///     The physical address.
+///
+pub inline fn virtToPhys(virt: var) @typeOf(virt) {
```

Comment:
not sure about pub inline. Porbs don't need the inline

Reply:
Good idea, this was actually why the virtToPhys tests used to fail on release-fast and why I ended up needing a separate `ADDR_OFFSET` variable, so I have now removed it.

Reply:
Ok scratch that, it was actually failing on release-safe so ignore all of the above :p